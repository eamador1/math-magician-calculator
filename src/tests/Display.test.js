import React from 'react';
import { render } from '@testing-library/react';
import Screen from '../components/Display';

describe('Screen Component', () => {
  it('renders input', () => {
    const { getByText } = render(<Screen input="1" />);

    // Use React Testing Library to check if the component renders the input text correctly
    const screenElement = getByText('1');

    // Assert that the screen element is in the document

    expect(screenElement).toBeInTheDocument();
    expect(screenElement).toMatchSnapshot();
  });
  it('renders input', () => {
    const { getByText } = render(<Screen input="99.9" />);

    // Use React Testing Library to check if the component renders the input text correctly
    const screenElement = getByText('99.9');

    // Assert that the screen element is in the document

    expect(screenElement).toBeInTheDocument();
    expect(screenElement).toMatchSnapshot();
  });
});
