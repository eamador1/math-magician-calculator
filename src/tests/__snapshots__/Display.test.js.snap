// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen Component renders input 1`] = `
<div
  class="input"
>
  1
</div>
`;

exports[`Screen Component renders input 2`] = `
<div
  class="input"
>
  99.9
</div>
`;
